From 6f9f0d433133477b9fdaca886ebad13ecbf7f543 Mon Sep 17 00:00:00 2001
From: Paul Keith <javelinanddart@gmail.com>
Date: Sun, 31 Dec 2017 02:25:55 +0100
Subject: [PATCH 14/19] klte-common: power: Update power hal extension for new
 qti hal

* And fix the LOG_TAG while we're at it

Change-Id: I81678e10385fef4c6881c17813f8ffa89caa2629
---
 power/power_ext.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/power/power_ext.c b/power/power_ext.c
index 60526ea..0506b1c 100644
--- a/power/power_ext.c
+++ b/power/power_ext.c
@@ -18,7 +18,7 @@
 #include <string.h>
 #include <fcntl.h>
 
-#define LOG_TAG "PowerHAL_H_Ext"
+#define LOG_TAG "PowerHAL_K_Ext"
 #include <utils/Log.h>
 
 /* touchkeys */
@@ -47,7 +47,7 @@ static void sysfs_write(char *path, char *s) {
     close(fd);
 }
 
-void cm_power_set_interactive_ext(int on) {
+void power_set_interactive_ext(int on) {
     ALOGD("%s: %s input devices", __func__, on ? "enabling" : "disabling");
     sysfs_write(TK_POWER, on ? "1" : "0");
     sysfs_write(TS_POWER, on ? "1" : "0");
-- 
2.7.4

